aquality-appium-mobile-template-cucumber/src/test/java/aquality/appium/mobile/template/cucumber/objectfactory/CustomObjectFactory.java

```diff
@@ -0,0 +1,36 @@
+package aquality.appium.mobile.template.cucumber.objectfactory;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import cucumber.api.guice.CucumberModules;
+import cucumber.runtime.java.guice.ScenarioScope;
+import io.cucumber.core.backend.ObjectFactory;
+
+public class CustomObjectFactory implements ObjectFactory {
+
+    private Injector injector;
+
+    public CustomObjectFactory() {
+        this.injector = Guice.createInjector(CucumberModules.createScenarioModule(), new ServiceModule());
```

Comment:
Should we just add MobileModule additionally here?
this way we can resolve all aquality services

Reply:
Do we really need it? Actually, it'd be better to use DI in all our templates. Let's discuss.